### ISSUE TITLE
Research: Sphinx

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#2]
## User story
1. As a team
2. We need an easily maintainable documentation system for the code
3. So we can easily generate high quality documentation

## Acceptance criteria
* Do some research on [Sphinx](https://www.sphinx-doc.org/en/master/)
* Also do some research on other python documentation systems and decide whether there is something better than Sphinx for our project

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
